### DISCUSSION TITLE
Setting power higher than 22dBm on LLCC68

### DISCUSSION THREAD

**Discussion by maalos** [jgromes/RadioLib#1359] [Q&A] [answered]
Hello. I have an E220-400M-30S (30dBm), however it fails to initialize from the LLCC68 class and I have to use SX1262 instead, but it has lower max power - only 22dBm. Is there a way around that? - for example setting the power via god mode or adding an extra check for the LLCC68 initializer which checks if the module reports itself as an SX1262?

**Comment by maalos** [comment]
it's kind of similar to this line:
https://github.com/jgromes/RadioLib/blob/c37015ef9467404d47261f8f4e737aadc8049009/src/modules/SX126x/SX1262.h#L15
but with LLCC68 modules it seems like some of them report themselves as LLCC68 and some as SX1262 (like mine)

**Comment by jgromes** [comment, marked as answer]
The maximum output power of a standalone LLCC68 is 22 dBm. Your module has an external PA that cannot be controlled by RadioLib. There's a [wiki page](https://github.com/jgromes/RadioLib/wiki/High-power-Radio-Modules-Guide) (and many other issues and discussions) about this.
